### ISSUE TITLE
Column sizing for "Compare Ingress" recipe

### ISSUE THREAD

**Issue by CapKenR** [walmartlabs/kubeman#5]
For the "Compare Ingress" recipe, the two comparison columns are not sized evenly. The left column remains small while the right column changes based on the window size. (See the attached for an example.)

In the "Compare Namespace Deployments" and "Compare Namespace Services" recipes, the two columns are sized evenly.

![Compare-Ingress-column-sizing](https://user-images.githubusercontent.com/11944619/68216442-1f816480-ff9e-11e9-97f4-fbcaad805e1c.png)

**Comment by uk000** [comment]
Yeah, chrome renderer performs dynamic column sizing based on the content, overriding the requested default widths. I need to look into adding some more CSS rules there. Perils of building a generic rendering engine for all content :)
